### ISSUE TITLE
TypeScript definitions excluded in .npmignore

### ISSUE THREAD

**Issue by omichelsen** [pburtchaell/redux-promise-middleware#236]
First of all, thanks for including TypeScript definitions! 

Currently they don't work however, since package.json is pointing to the `src` directory which is excluded by .npmignore:

```
"types": "src/index.d.ts"
```

Applies to the current v6.

**Comment by pburtchaell** [comment]
Hi @omichelsen, good catch. I opened #237, which should fix this, and published a fix to npm. Can you install the beta and let me know if it works for you?

```
npm install redux-promise-middleware@6.0.1-beta.0
```

**Comment by omichelsen** [comment]
Works like a charm 👍

**Comment by pburtchaell** [comment]
Great! Will release this at some point today or in the next couple days.

**Comment by pburtchaell** [comment]
Merged to master and released to npm as `redux-promise-middleware@6.0.1`.
